integration-test/src/test/java/org/sagebionetworks/IT049FileHandleTest.java

```diff
@@ -558,6 +558,39 @@ public void testMultipartUploadV2Reset() throws FileNotFoundException, SynapseEx
 		assertFalse(startStatus.getUploadId().equals(statusAgain.getUploadId()));
 	}
 
+	@Test
+	public void testCreateExternalS3FileHandleFromExistingFile() throws Exception {
+		assertNotNull(imageFile);
+		assertTrue(imageFile.exists());
+		String md5 = MD5ChecksumHelper.getMD5Checksum(imageFile);
+
+		// Upload the owner.txt to S3 so we can create the external storage location
+		String baseKey = "integration-test/IT049FileHandleTest/testCreateExternalS3FileHandleFromExistingFile/" + UUID.randomUUID().toString();
+		String key = baseKey + FILE_NAME;
+		uploadOwnerTxtToS3(config.getS3Bucket(), baseKey, synapse.getUserProfile(userToDelete.toString()).getUserName());
+
+		// upload the little image to S3, but not through Synapse
+		synapseS3Client.putObject(config.getS3Bucket(), key, imageFile);
+
+		// Create the storage location setting
+		ExternalS3StorageLocationSetting storageLocationSetting = new ExternalS3StorageLocationSetting();
+		storageLocationSetting.setBucket(config.getS3Bucket());
+		storageLocationSetting.setBaseKey(baseKey);
+		storageLocationSetting.setUploadType(UploadType.S3);
+		storageLocationSetting = synapse.createStorageLocationSetting(storageLocationSetting);
+
+		S3FileHandle fh = new S3FileHandle();
+		fh.setStorageLocationId(storageLocationSetting.getStorageLocationId());
+		fh.setBucketName(config.getS3Bucket());
+		fh.setKey(key);
+		fh.setContentMd5(md5);
+
+		S3FileHandle result = synapse.createExternalS3FileHandle(fh);
+		assertNotNull(result);
+		toDelete.add(result);
+		assertNotNull(result.getFileName());
+	}

```

Comment:
Prior to this PR, there was no integration test for creating an S3FileHandle using an existing file in S3